README.md

```diff
@@ -38,15 +37,41 @@ slack-handler.json
         "timeout": 30,
         "filters": [
             "is_incident"
-        ]
+        ],
+        "runtime_assets": ["sensu-slack-handler_linux_amd64"]
```

Comment:
Let's check that `asset add` produces an asset name like this one... I hope it is installing multi-build asset definitions with simpler names...